test/eredis_common.erl

```diff
@@ -0,0 +1,8 @@
+-module(eredis_common).
+
+-export([ get_tcp_ports/0 ]).
+
+% Get number of available TCP ports (includes TLS)
+get_tcp_ports() ->
+    [Port || Port <- erlang:ports(),
+             {name, "tcp_inet"} =:= erlang:port_info(Port, name)].
```

Comment:
Nice! I didn't know you could do this.

But the module name "eredis_common"? What is it common for? Since it's just for the tests, I'd call it eredis_test_util or something like that.